src/solvers/prop/decision_procedure_assumptions.cpp

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************\
+
+Module: Decision procedure with incremental solving under assumptions
+
+Author: Peter Schrammel
+
+\*******************************************************************/
+
+#include "decision_procedure_assumptions.h"
+
+void decision_procedure_assumptionst::set_assumptions(const bvt &)
+{
+  UNREACHABLE;
```

Comment:
Could this be pure virtual or something else?  Base implementations of methods that are just "nope!" make me kinda nervous.

Reply:
True, I've actually thought I had patched that. Will change.

Reply:
Done